s30130-EmilGulaj/pamietnik.txt

```diff
@@ -0,0 +1,68 @@
+2024-03-10
+Moj kochany pamietniczku:
+
+Opis zajec
+	To byly zajecia co prawda wprowadzajace do samego przedmiotu, 
+	jak i do pierwszego zadania ktore było omawiane na wykladzie,
+	ale i tak były ciekawe.
+
+Czego sie nauczyles
+	Dowiedzialem sie jak bardzo duzo nie wiem, i jak to czego nie wiem 
+	jest mi potrzebne.
+
+Co jest dla Ciebie jeszcze nie jasne i musisz poglebic temat
+	Zdecydowanie musze nauczyc sie tego jak dziala Git/Github,
+	bo nie mam o tym zielonego pojecia, a wydaje sie byc bardzo
+	uzytecznym narzedziem. 
+
+Co bylo zaskakujacego
+	Przedewszystkim zaskoczylo mnie to, ze wiedza na temat gita/linuxa,
+	jest taka istotna by chociazby otrzymac pierwsza prace, ale
+	gdy sie nad tym zastanowilem, rzeczywiscie ma to sens.
+
+Twoje przemyslenia o calych zajeciach i zadaniach
+	Zajecia wydaja sie byc jedne z ciekawszych na tej uczelni, i podoba
+	mi sie to jak na kazdym momencie jesteśmy wrzucani na głeboka wode.
+
+
+2024-03-24
+Moj kochany pamietniczku:
+
+Opis zajec
+	Na tych zajeciach zajmowalismy sie pisaniem podstawowych
+	skryptow w bashu.
+
+Czego sie nauczyles
+	Nauczylem sie jak robic podstawowe skrypty, oraz wielu
+	ciekawych rzeczy na temat wykorzystania basha w zyciu
+	codziennym.
+
+Co jest dla Ciebie jeszcze nie jasne i musisz poglebic temat
+	Zdecydowanie musze się jeszcze sporo nauczyć na temat skryptów
+	a najbardziej interesuje mnie tworzenie ich w np. pythonie.
```

Comment:
Pythona jest bardzo łatwo się nauczyć jeśli opanujesz składnię jakiegoś języka ze składnią C-podobną więc C/C++/Java/C#/PHP wszystkie pasują. Kolejne języki to już tylko kwestia zrozumienia składni i możliwości... no chyba, że przechodzisz z języka imperatywnego na funkcyjny... wtedy jest trudniej.